src/member/views.py

```diff
@@ -590,6 +590,33 @@ def save_referent_information(self, client, billing_member, emergency):
                 lastname=referent_information.cleaned_data.get("lastname"),
             )
             referent.save()
+            ref_email = referent_information.cleaned_data.get(
+                "email", None)
+            ref_work_phone = referent_information.cleaned_data.get(
+                "work_phone", None)
+            ref_cell_phone = referent_information.cleaned_data.get(
+                "cell_phone", None)
+            if ref_email:
+                ctc_email = Contact.objects.create(
+                    type=EMAIL,
+                    value=ref_email,
+                    member=referent,
+                )
+                ctc_email.save()
+            if ref_work_phone:
+                ctc_work_phone = Contact.objects.create(
+                    type=WORK,
+                    value=ref_work_phone,
+                    member=referent,
+                )
+                ctc_work_phone.save()
+            if ref_cell_phone:
+                ctc_cell_phone = Contact.objects.create(
+                    type=CELL,
+                    value=ref_cell_phone,
+                    member=referent,
+                )
+                ctc_cell_phone.save()
```

Comment:
1. You don't need to call save() when using create()

2. You should use the method `add_contact_information`, provided by the Member model: https://github.com/savoirfairelinux/sous-chef/blob/dev/src/member/models.py#L134.